### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by TylrDn**
Added a new logo image and updated the README to include it at the top.

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Best practice is to use lowercase filenames with no spaces

Also the logo is way too big.

**Comment by jbampton** [comment]
You can use a "-" in the filename instead of spaces

**Review by jbampton — CHANGES_REQUESTED** [review]
You now have two logo files

**Comment by TylrDn** [comment]
I see now I have multiple '.png' assets. Should I resize the image or use an HTML Exception?

**Comment by jbampton** [comment]
Let us keep the current image size and use HTML width 150.

I want to work on this after you are finished.  Another PR for me.

We can always change it later.  Let us see

**Comment by TylrDn** [comment]
Is there anything else I need to do in order to close out my first PR on this topic?

This is my first so bear with me. 2 uncussessful checks:

The configuration file (path: .github/labeler.yml) was not found locally, fetching via the api
Error: Error: found unexpected type for label 'documentation' (should be array of config options)
Error: found unexpected type for label 'documentation' (should be array of config options)

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
 documentation:
```
Comment (jbampton) [inline .github/labeler.yml:1]:
```suggestion
---
documentation:
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+<!-- markdownlint-disable MD033 -->
```
Comment (jbampton) [inline README.md:1]:
```suggestion
```

**Review by jbampton — APPROVED** [review]

**Comment by jbampton** [comment]
Thank you :+1:

**Pull request merged**